manifests/web-terminal.clusterserviceversion.yaml

```
@@ -119,8 +119,10 @@ spec:
     5. Remove lingering service, secrets, and configmaps
 
             kubectl delete all --selector app.kubernetes.io/part-of=devworkspace-operator,app.kubernetes.io/name=devworkspace-webhook-server
-            kubectl delete serviceaccounts devworkspace-webhook-server -n openshift-operators
-            kubectl delete configmap devworkspace-controller -n openshift-operators
+            kubectl delete serviceaccounts devworkspace-webhook-server -n openshift-operators --ignore-not-found=true
+            kubectl delete configmap devworkspace-controller -n openshift-operators --ignore-not-found=true
+            kubectl delete clusterole devworkspace-webhook-server -n openshift-operators --ignore-not-found=true
```

seems r is missing in `clusterole`, and cluster role is cluster-scoped
```suggestion
            kubectl delete clusterrole devworkspace-webhook-server --ignore-not-found=true
```

---

pushed fix